.circleci/config.yml

```diff
@@ -11,35 +12,9 @@ workflows:
           context: Danger
       - kitchen/lint:
           name: lint
-      - kitchen/dokken-single:
-          name: server-debian-8
-          suite: server
-          platform: debian-8
-          requires: [danger, lint]
-      - kitchen/dokken-single:
-          name: server-debian-9
-          suite: server
-          platform: debian-9
-          requires: [danger, lint]
-      - kitchen/dokken-single:
-          name: server-centos-7
-          suite: server
-          platform: centos-7
-          requires: [danger, lint]
-      - kitchen/dokken-single:
-          name: server-fedora-latest
-          suite: server
-          platform: fedora-latest
-          requires: [danger, lint]
-      - kitchen/dokken-single:
-          name: server-ubuntu-1604
-          suite: server
-          platform: ubuntu-1604
-          requires: [danger, lint]
-      - kitchen/dokken-single:
-          name: server-ubuntu-1804
+      - kitchen/dokken:
+          name: server
```

Comment:
Ah, much nicer than all that boilerplate.

Reply:
Though it does run it in serial, so it's a bit slower.